frontend/src/common/authentication/auth-walls/BCeIDAuthWall.tsx

```diff
@@ -35,7 +38,8 @@ export const BCeIDAuthWall = ({
 
   const { userRoles, companyId, isNewBCeIDUser } =
     useContext(OnRouteBCContext);
-
+  const { data: companyInfo } =
+    useCompanyInfoDetailsQuery(getDefaultRequiredVal(0, companyId));
```

Comment:
You don't have to query company details. use the context instead. It should have isSuspended field.